src/App.js

```diff
@@ -1,25 +1,11 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
```

Comment:
- [ ] The import statement is wrong. It should be `'react'` instead of `'reaact'`. Kindly correct the typographical error. 👍 😎

Reply:
I see 'react instead of the 'reaact' which you talk about

Reply:
@Gambit142, based on your comment above I think I will just remove the import react statement completely.